modules/storage-persistent-storage-pv.adoc

```diff
@@ -60,6 +60,7 @@ ifndef::openshift-rosa[]
 endif::openshift-rosa[]
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 - IBM VPC Block
+- {ibmpowerProductName} Virtual Server Block
```

Comment:
Small suggestion: it seems like this list tries to be in alphabetical order (with the exception of the GCP items). If there's no authoritative order that this list is otherwise following, I'd consider moving this list item to be above "IBM VPC Block" to maintain alphabetical order.